pkg/core/resources/apis/mesh/validators.go

```diff
@@ -168,6 +168,30 @@ func ProtocolValidator(protocols ...string) SelectorValidatorFunc {
 	}
 }
 
+// Resource is considered valid if it pass validation of any message
+func ValidateAnyResourceYAML(resYAML string, msgs ...proto.Message) error {
```

Comment:
I think when it comes to processing the collections **Any** means "at least one element is true". But if I'm correct this method is considered valid when **All** messages are valid. So I guess it should be either renamed or logic should be changed

Reply:
No, this method is considered valid when *Any* message is valid

Reply:
ok, you are right 🤕  changed